docs/index.html

```diff
@@ -31,6 +31,17 @@
                    <div class="content-data">
 
 <h1 id="crbm-viz">CRBM-Viz</h1>
+<h2 id="current-build">Current Build</h2>
+<p><a href="https://crbm-viz.herokuapp.com/">https://crbm-viz.herokuapp.com/</a></p>
+<h2 id="documentation">Documentation</h2>
+<p><a href="https://reproducible-biomedical-modeling.github.io/CRBM-Viz/">https://reproducible-biomedical-modeling.github.io/CRBM-Viz/</a></p>
```

Comment:
<a href="https://reproducible-biomedical-modeling.github.io/CRBM-Viz/">reproducible-biomedical-modeling</a>
Instead of the full link?